### PR TITLE
Update docs with tracking id parameter

### PR DESCRIPTION
**Pull request by exaroth**
Update docs with tracking requests information

### CODE REVIEW

File: docs/users/advanced.rst [hunk 1]
```diff
@@ -1135,6 +1135,33 @@ you need to pass those parameters back to us::
...
+=====================
+
+.. _setting_tracking_id:
+
+When instantiating Client object it's also possible to set optional tracking ID 
+that will be set with each request performed by `make_request` method::
+
+    >>> from pyticketswitch import Client
+    >>> client = Client('example-user', 'some-password', tracking_id='xyz')
+    >>> client.make_request('/get-data', {})
```
Comment (nicwest) [inline docs/users/advanced.rst]: instead of `/get-data` do `test.v1` or something like that.

File: docs/users/advanced.rst [hunk 2]
```diff
@@ -1135,6 +1135,33 @@ you need to pass those parameters back to us::
...
+
+This is useful when using common id for tracking requests across the
```
Comment (nicwest) [inline docs/users/advanced.rst]: a common id

Comment (exaroth) [inline docs/users/advanced.rst]: 9d5f487 Fixed

**Comment by nicwest** [comment]
LGTM

**Pull request merged**